### PR TITLE
fix: paginate styles

### LINKED ISSUE #203: Paginator overrides content / el paginador se sobre pone al contenido
**Describe the bug**
al ingresar al el aplicativo, ingreso al menu lateral a proveedores, alli puedo en la lista de porveedores que el paginador queda sobre el contenido que ahi en la pagina.

When entering the application, I enter the side menu to providers, there I can in the list of providers that the locator is in the content that is on the page.

**To Reproduce**

1. login
2. Side menu
3. Suppliers

**Expected behavior**

Al ingresar a esta vista y cargar los datos, el paginador debe estar en la parte inferior y no superponerse al contenido.

When entering this view and loading the data, the paginator should be at the bottom and not overlap the content.

- Have as a reference the design that is in adobe XD

https://xd.adobe.com/view/c3c8d408-317c-4f89-bd90-094f636b567a-969b/screen/6d5aa4dd-5c0a-4c9c-929f-d0f6346343f1/
imagen

**Screenshots**
imagen

**Additional context**

### PR DESCRIPTION
**Pull request by Eghinner**
chrome_i5wAat2iye

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

File: components/Paginated/Paginated.module.scss [hunk 1]
```diff
@@ -20,16 +21,11 @@
 
 .paginated-container {
   align-items: center;
-  bottom: 0;
   display: flex;
   justify-content: space-between;
-  left: 0;
   list-style-type: none;
-  margin: 30px 50px;
+  margin: auto 50px 30px;
```
Comment (heliomar-pena) [inline components/Paginated/Paginated.module.scss:27]: I though that `margin-top: auto` not works, it give the same result if margin-top is `0`?

imagen

File: components/Paginated/PaginatedSuppliers.tsx [hunk 2]
```diff
@@ -26,46 +26,44 @@ const PaginatedSuppliers: FC<PaginatedSuppliersProps> = ({
...
+        <label>
+          {slicedProviders.length * CurrentPage} of {totalProvidersLength}
+        </label>
       </div>
+      <ul>
+        <div
+          className={styles.button}
+          onClick={() => {
+            if (CurrentPage !== 1) setCurrentPage(CurrentPage - 1)
+          }}
```
Comment (heliomar-pena) [inline components/Paginated/PaginatedSuppliers.tsx]: what about:

```suggestion
          onClick={() => CurrentPage !== 1 && setCurrentPage(CurrentPage - 1)}
```

File: components/Paginated/PaginatedSuppliers.tsx [hunk 3]
```diff
@@ -26,46 +26,44 @@ const PaginatedSuppliers: FC<PaginatedSuppliersProps> = ({
...
+        </div>
+        <div
+          className={styles.button}
+          onClick={() => {
+            if (CurrentPage !== pageNumber) setCurrentPage(CurrentPage + 1)
+          }}
+        >
+          <span>Siguiente</span>
+        </div>
+      </ul>
```
Comment (heliomar-pena) [inline components/Paginated/PaginatedSuppliers.tsx]: what about:

```suggestion
          {pageNumbers?.map(page =>
              CurrentPage === page ? (
                <li className={styles.active} key={page} onClick={() => setCurrentPage(page)}>
                  {page}
                </li>
              ) : (
                <li key={page} onClick={() => setCurrentPage(page)}>
                  {page}
                </li>
              ),
            )}
        </div>
        <div
          className={styles.button}
          onClick={() => {
            if (CurrentPage !== pageNumber) setCurrentPage(CurrentPage + 1)
          }}
        >
          <span>Siguiente</span>
        </div>
      </ul>
```

File: components/Paginated/PaginatedSuppliers.tsx [hunk 4]
```diff
@@ -26,46 +26,44 @@ const PaginatedSuppliers: FC<PaginatedSuppliersProps> = ({
...
+            pageNumbers.map(page =>
+              CurrentPage === page ? (
+                <li className={styles.active} key={page} onClick={() => setCurrentPage(page)}>
+                  {page}
+                </li>
+              ) : (
+                <li key={page} onClick={() => setCurrentPage(page)}>
+                  {page}
+                </li>
+              ),
```
Comment (heliomar-pena) [inline components/Paginated/PaginatedSuppliers.tsx]: What about 

```suggestion
                <li className={CurrentPage === page ? styles.active : ""} key={page} onClick={() => CurrentPage !== page && setCurrentPage(page)}>
                  {page}
                </li>
```

**Review by heliomar-pena — APPROVED** [review]
good work! just some comments

File: components/Paginated/Paginated.module.scss (hunk 1 above)
Comment (Eghinner) [reply components/Paginated/Paginated.module.scss:27]: Im not sure, maybe in flex block works depending on the space

Comment (heliomar-pena) [reply components/Paginated/Paginated.module.scss:27]: I'll investigate it

**Comment by Israel-Laguan** [comment]
@angelik0828 please check

**Comment by angelik0828** [comment]
## The other task was approved since it should not move with the scroll, it should be fixed on the page.

**Pull request closed without merging**